Sources/web3swift/Utils/ENS/ENSResolver.swift

```diff
@@ -66,6 +66,11 @@ public extension ENS {
             return supports
         }
 
+        public func supportsInterface(interfaceID: InterfaceName) async throws -> Bool {
+            guard let supports = try? await supportsInterface(interfaceID: interfaceID.hash) else {throw Web3Error.processingError(desc: "Can't get answer")}
+            return supports
+        }
```

Comment:
nice one 👌

Reply:
Please, add a few spaces in the `else` clause around the error that is thrown.
Even though in other places in the library you see similar formatting I'd suggest moving away from it as it makes the code look bit strange. There is no benefit in removing spaces.

Reply:
That's why I propose we enable `swiftlint` now for PR's.